### PR TITLE
[FEC-197] New test data models CLI improvements

### PR DESCRIPTION
**Pull request by CarlosCortizasCT**
In this PR we include two main improvements:
* The CLI will ask the user if they want to generate a preset example for the new model
* The CLI will ask the user if the new model requires a draft and will generate its files if needed

https://github.com/user-attachments/assets/5bf238a9-71cf-4d85-a549-3f4ea50dfc04

### CODE REVIEW

**Review by ddouglasz — APPROVED** [review]

**Review by nima-ct — APPROVED** [review]

File: generators/src/new-test-model/new-test-model.generator.ts [hunk 1]
```diff
@@ -103,19 +134,46 @@ export const newTestModelGenerator: CodeGenerator = {
...
+              isPresetExampleRequired,
+              modelName: `${modelName}Draft`,
+              modelCodename: `${modelCodename}-draft`,
+              graphqlTypePrefix,
+            })
+          );
+        });
+    }
 
     console.log('\nAll set! 🚀');
```
Comment (tdeekens) [inline generators/src/new-test-model/new-test-model.generator.ts:178]: Nit
```suggestion
    console.log('\n 🚀 All set! We've generated all files. You can now adjust them to your needs.');
```

**Review by tdeekens — APPROVED** [review]
Super cool. Only nits. Take them or leave them :)

File: generators/src/new-test-model/new-test-model.generator.ts [hunk 2]
```diff
@@ -103,19 +134,46 @@ export const newTestModelGenerator: CodeGenerator = {
...
+        .filter(modelTemplatesFilter(isPresetExampleRequired, true))
+        .forEach((template) => {
+          const filePath = join(
+            outputPath,
+            'src',
+            `${modelCodename}-draft`,
+            template.templatePath
+          );
+          ensureDirectory(filePath);
+          console.log(`Generating file: ${filePath}`);
```
[the remark this answers is not in the document]
Comment (CarlosCortizasCT) [reply generators/src/new-test-model/new-test-model.generator.ts:177]: Yes, it is intended.

File: generators/src/new-test-model/new-test-model.generator.ts [hunk 3]
```diff
@@ -103,19 +134,46 @@ export const newTestModelGenerator: CodeGenerator = {
...
+    modelTemplatesData
+      .filter(modelTemplatesFilter(isPresetExampleRequired))
+      .forEach((template) => {
+        const filePath = join(
+          outputPath,
+          generationType === 'standalone' ? 'src' : '',
+          template.templatePath
+        );
+        ensureDirectory(filePath);
+        console.log(`Generating file: ${filePath}`);
```
[the remark this answers is not in the document]
Comment (CarlosCortizasCT) [reply generators/src/new-test-model/new-test-model.generator.ts:154]: Yes, it is intended.

File: generators/src/new-test-model/new-test-model.generator.ts (hunk 1 above)
Comment (CarlosCortizasCT) [reply generators/src/new-test-model/new-test-model.generator.ts]: Updated here: 2f5246d85c5cabaeb2e8730c6e9929881d859dce

File: generators/src/new-test-model/new-test-model.generator.ts [hunk 4]
```diff
@@ -31,6 +46,20 @@ export const newTestModelGenerator: CodeGenerator = {
     });
     const modelCodename = snakeCase(modelName).replaceAll('_', '-');
 
+    const { isDraftRequired } = await prompts({
+      type: 'confirm',
+      name: 'isDraftRequired',
+      message: 'Does this model require a Draft?',
```
[the remark this answers is not in the document]
Comment (CarlosCortizasCT) [reply generators/src/new-test-model/new-test-model.generator.ts:55]: To be honest, I don't see too much of a difference.
If the concern is that users won't know what we mean by Draft, I would prefer to get feedback from them first.

[the remark this answers is not in the document]
Comment (tdeekens) [reply generators/src/new-test-model/new-test-model.generator.ts:55]: Sounds good. Fair to me.

**Comment by tdeekens** [comment]
Thanks for the adjustments @CarlosCortizasCT!

**Pull request merged**